src/hooks/member.ts

```diff
@@ -520,7 +520,7 @@ export const useMemberNotesAdmin = (
               return {
                 ...prev,
                 member_note_aggregate: fetchMoreResult.member_note_aggregate,
-                member_note: [...prev.member_note, ...fetchMoreResult.member_note],
+                member_note: [...(prev ? prev.member_note : []), ...fetchMoreResult.member_note],
```

Comment:
NITS: may modify the `prev` 
ex:
```
let newPrev = prev ? prev.member_note : []

return {
   ...newPrev,
   // do something ....
}
```

Reply:
![image](https://user-images.githubusercontent.com/28040359/148508568-d530e558-bd5b-41f1-afa3-519fa62cd6b4.png)

Reply:
![image](https://user-images.githubusercontent.com/20295628/148512985-7fadd0ea-a153-4bd7-acc8-a6e895826c25.png)

Reply:
sorry, this is what I mean